src/js/components/SharedComponents/CalendarRangeDatePicker.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { PropTypes } from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+
+import Moment from 'moment';
+
+function Navbar({
+  onPreviousClick,
+  onNextClick,
+  className
+}) {
+  const styleLeft = {
```

Comment:
Inline styles should be avoided where possible. Could you use a modifier class and add these properties with CSS?